web/packages/teleterm/src/ui/Search/pickers/ResultList.tsx

```diff
@@ -139,27 +139,26 @@ export const NonInteractiveItem = styled.div`
   }
 
   :not(:last-of-type) {
-    border-bottom: 1px solid
-      ${props => props.theme.colors.levels.surfaceSecondary};
+    border-bottom: 1px solid ${props => props.theme.colors.spotBackground[0]};
   }
 
   padding: ${props => props.theme.space[2]}px;
   color: ${props => props.theme.colors.text.contrast};
-  background: ${props => props.theme.colors.levels.surface};
+  background: inherit;
 `;
 
 const InteractiveItem = styled(NonInteractiveItem)`
   cursor: pointer;
 
   &:hover,
   &:focus {
-    background: ${props => props.theme.colors.levels.elevated};
+    background: ${props => props.theme.colors.spotBackground[0]};
```

Comment:
@rudream What's the "semantic" difference between `levels.popout` and `spotBackground`, between `spotBackground[0]` and `spotBackground[2]`?

I feel like we should use the elevation scheme all the way through, `spotBackground` feels like an escape hatch for when it was inconvenient to use `levels`. 🤔

@gzdunek But I think in this case we could just use `levels.popout` instead of `spotBackground[0]` here, couldn't we?

Reply:
Levels are for large areas and are solid colors. 

Spot backgrounds are for smaller elements, are mostly transparent so they can add show a change within elements that sit atop the current level. As such, they are most frequently used for interactivity, because their transparency allows them to work across every single level. 

Levels should not be used to designate interactivity—they're the broad backgrounds used to hold other elements. If we used levels for interactivity, we would need to have unnecessarily complex logic for calibrating interactive effects for things like minimal buttons depending on what level we're placing them in. With spot backgrounds, we can say (for example) that a minimal button always gets spot[0] for a hover state and spot[1] for an active state, regardless of the level.

See [the Figma documentation for Bloomberg](https://www.figma.com/file/Gpjs9vjhzUKF1GDbeG9JGE/Application-Design-System?node-id=7020%3A33956&t=bHgvlFIZDHEtaZHl-1) for a glimpse how this works in practice.

Reply:
Thanks for the detailed explanation Kenny!

It looks like the included link is incorrect :)

Reply:
> It looks like the included link is incorrect :)

It's correct, I think Kenny meant "look at _our_ Figma documentation we made for a customer". ;)

Reply:
But by doing this we will apply hover also to the input from the "open state", I've just tried that and it doesn't look good :/